dspace-server-webapp/src/main/java/org/dspace/app/rest/scripts/handler/impl/RestDSpaceRunnableHandler.java

```diff
@@ -186,10 +186,6 @@ public Process getProcess() {
             return processService.find(context, processId);
         } catch (SQLException e) {
             log.error("RestDSpaceRunnableHandler with process: " + processId + " could not be found", e);
-        } finally {
-            if (context.isValid()) {
-                context.abort();
-            }
```

Comment:
I'm confused about this change. Why did we remove the aborting of this new Context?  It seems like we are now leaving a Context open in this method.

Reply:
Just realized this question wasn't answered either.  Looking at this method, my best guess is you might need the returned Process to be attached to an active Context.  If that's the case, this `getProcess()` method likely should take a Context object as a parameter.  Otherwise, this Context may never be closed, which could result in odd behaviors.